api/services/ocean/spark/paths/workspaces.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Ocean Spark
+
+post:
+  summary: "Create Workspace"
+  description: |
+    Create Ocean Spark Workspace
+  operationId: "OceanSparkWorkspaceCreate"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../../../../commons/parameters/accountId.yaml"
+  requestBody:
+    required: false
+    content:
+      application/json:
+        schema:
+          allOf:
+            - $ref:  "../schemas/oceanSparkWorkspaceCreateWrapper.yaml"
+            - type: "object"
+              properties:
+                workspace:
+                  type: object
+                  required:
+                    - name
+  responses:
+    200:
+      $ref: "../responses/workspaces.yaml"
+    400:
+      description: "Bad Request"
+
+get:
+  summary: "List Workspaces"
+  description: >
+    List the Ocean Spark workspaces in the specified cluster.
+  operationId: "OceanSparkWorkspaceList"
+  tags:
+    - "Ocean Spark"
+  parameters:
```

Comment:
I don't think the parameters are accurate here.
we should only have accountId and clusterId if I'm not wrong